validation/src/test/scala/hmda/validation/engine/lar/quality/LarQualityEngineSpec.scala

```diff
@@ -0,0 +1,21 @@
+package hmda.validation.engine.lar.quality
+
+import hmda.parser.fi.lar.LarGenerators
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+class LarQualityEngineSpec
+    extends PropSpec
+    with PropertyChecks
+    with MustMatchers
+    with LarGenerators
+    with LarQualityEngine {
+
+  property("A LAR must pass quality checks") {
+    forAll(larGen) { lar =>
+      whenever(lar.id == 2) {
+        checkQuality(lar).isSuccess mustBe true
+      }
+    }
+  }
```

Comment:
I think I agree with you.  Previously, I know that I've had to change the `LarGenerators` to match our tests.  I imagine that's what we'll continue to do: whenever we find a quality check that will make the generators fail, we can just modify the generators.